.travis.yml

```diff
@@ -11,6 +11,11 @@ cache:
   directories:
     - node_modules
 
+before_install:
+  # Install yarn
+  - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.15.2
+  - export PATH=$HOME/.yarn/bin:$PATH
```

Comment:
Will add this to app-kit.

Reply:
We have it already there, you can bump the yarn version though 😉

Reply:
Did that.